flaskr/templates/base.html

```diff
@@ -10,7 +10,7 @@
     <nav>
 	  <ul class="menu">
 	    <li class="logo"><a href="{{ url_for('index')}}">StockMeetsBagel</a></li>
-	    <li class="item" id="about"><a href="#">About</a></li>
```

Comment:
Heads up: Jessica added an 'About' page in the `ui-work` branch.

Reply:
oh whoops i'll add it back